### ISSUE TITLE
My password changes

### ISSUE THREAD

**Issue by ignSapser** [sp614x/optifine#1616]
Hey, I donated for a Optifne Cape for over 2 years ago, and I have experienced this issue meanwhile. I usually change stuff (skin, name etc.) because I easily gets tired on the old stuff, so I decided to change how my Optifine Cape should be designed. I watched videos on how to and found out it was pretty simple, just like that I changed my cape design. When I, later on, tried logging into Minecraft it wouldn’t let me log in which I found very confusing since I was on Minecraft the second ago. I went to Mojang, Minecraft.net and Optifine’s own website and tried to log in which didn’t work. I basically couldn’t launch Minecraft or log in to Minecraft at all. I tried changing my password which lead me into contacting Mojag due issues which took me over a month. I might sound dramatic but at that time Minecraft was all I had. Since then I’ve never changed my Cape since I don’t want to go thru this again. Do I contact Mojang about this because optifine was the reason to this problem, all because of a Minecraft cape. I’m not struggling with the issue at the moment but for further actions I would love to know who to contact and what to do.

**Comment by iSkeptical** [comment]
Make sure to use different passwords for each website you use. Do not keep these stored on your computer. Make sure that your password isn't short or simple, like McBoy2001, but more like #9Bal!b*i%^_98. Make sure that you use strong security questions that ONLY you would know, meaning its not guessable. Make sure you don't have other tabs open, and always decline cookies on websites you do not know. You can further your online security by avoiding any malicious websites, like trying to download 1.13 forge (when it obviously doesn't exist yet), hacked clients, porn, or any cracked software. I'd search for any malware or spyware that you may have on your computer. As for contact, you will have to verify your identity with Mojang, and this issue probably has nothing to deal with optifine and your account is more than likely compromised. [Meaning whatever server you where on, someone may have pulled your username and email and their hack/software guessed your password.]

I hope this wall of text helped! :)

**Comment by TheAlienDrew** [comment]
Um... I actually just stumbled upon this post because it's right under mine when you search "email", but I think I know the problem. I literally has the EXACT same thing happen, at least with the email and password part for Minecraft. I wasn't using Optifine at the time (still setting up the new pc, downloading everything), but fairly recent. Anyways, apparently using a VPN of sorts actually makes it to where you can't sign into your Minecraft normally. With all other sites and apps I used, none had this problem, so I'm still clueless with why Minecraft does it.

So basically, if you recently got a new VPN (or are using any proxy's of sorts), try to disable those and logging in. (yes I know, sad days, can't use your handy ProtonVPN or the TOR networking, but works).

And to be clear once again, NOT CAUSED BY OPTIFINE.
